src/test/java/teammates/test/cases/datatransfer/FeedbackResponseAttributesTest.java

```diff
@@ -67,8 +72,12 @@ public void testBuilderWithDefaultValues() {
         assertEquals(Const.TIME_REPRESENTS_DEFAULT_TIMESTAMP, observedFeedbackResponseAttributes.getUpdatedAt());
     }
 
+    /*
```

Comment:
It is not true that all of the fields are non-null. For example, `responseMetaData` and `feedbackResponseId` can be null. There is some non-trivial work required to double check if there are any other fields that can be null otherwise we could have to some degree enforced this already. So maybe like the following instead?
```Java
/*
Not all possible states are tested because it is not known which fields are nullable.
However, it is at least known that `responseMetaData` and `feedbackResponseId` can be `null`.
*/
```
Anyway, this isn't a JavaDoc and shouldn't be anyway so let's not add the `*` prefix as it can be easily mistaken to be JavaDoc.

Reply:
So I'm wrong, `feedbackResponseId` cannot be null (and so let's prevent `FeedbackResponseAttributes` from having nullable `feedbackResponseId` too).